content/docs/guides/liquibase-workflow.md

```diff
@@ -0,0 +1,484 @@
+---
+title: Liquibase developer workflow with Neon
+subtitle: Implement a developer workflow with Liquibase and Neon branching
+enableTableOfContents: true
+---
+
+Liquibase is an open-source database-independent library for tracking, managing, and applying database schema changes. To learn more about Liquibase, refer to the [Liquibase documentation](https://docs.liquibase.com/home.html).
+
+This guide shows how to set up a developer workflow using Liquibase with Neon's branching feature. The workflow involves making schema changes to a database on a development branch and applying those changes back to the source database on the main branch of your Neon project.
+
+The instructions in this guide are based on the workflow described in the [Liquibase Developer Workflow](https://www.liquibase.org/get-started/developer-workflow) tutorial.
+
+## Prerequisites
+
+- A Neon account. See [Sign up](/docs/get-started-with-neon/signing-up).
+- A Neon project. See [Create your first project](/docs/get-started-with-neon/setting-up-a-project).
+- Liquibase requires Java. To check if you have Java installed, run `java --version`. If not, refer to the installation instructions for your operating system.
```

Comment:
Include link to java download? Or is it so common in this domain it would feel too hand-holding?

Reply:
Linked to the Liquibase Java requirements page, which has a download link and required version info.